package.json

```diff
@@ -4,13 +4,14 @@
   "scripts": {
     "codegen:rinkeby": "graph codegen -- subgraph-rinkeby.yaml",
     "build:rinkeby": "graph build -- subgraph-rinkeby.yaml",
-    "deploy:rinkeby": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- vbstreetz/witswap-staking-rinkeby subgraph-rinkeby.yaml",
+    "deploy:rinkeby": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- joyfulmagician/taraswap_staker-rinkeby subgraph-rinkeby.yaml",
     "codegen:mainnet": "graph codegen -- subgraph-mainnet.yaml",
     "build:mainnet": "graph build -- subgraph-mainnet.yaml",
-    "deploy:mainnet": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- vbstreetz/witswap-staking subgraph-mainnet.yaml"
+    "deploy:mainnet": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- joyfulmagician/taraswap_staker subgraph-mainnet.yaml",
+    "deploy:local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 taraswap_staker subgraph-mainnet.yaml"
```

Comment:
besides this we need a `create-local` command and a `remove-local` one. For reference please look into the `v3-subgraph` repo

Reply:
"create:local": "graph create --node http://localhost:8020/ taraswap_staker",
"remove:local": "graph remove --node http://localhost:8020/ taraswap_staker"